### PR TITLE
OpenAPI spec

### PR DESCRIPTION
**Pull request by SidDar360**

### CODE REVIEW

File: ingester.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+openapi: 3.0.3
+info:
+  title: Ingester
+  description: |-
+  termsOfService: http://swagger.io/terms/
```
Comment (peterinfra) [inline ingester.yaml:5]: As an example: change to a different URL

File: ingester.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  version: 1.0.11
+externalDocs:
+  description: Find out more about Swagger
+  url: http://swagger.io
+servers:
+  - url: https://ingester.swagger.io/api/v3
+tags:
+  - name: logs
+#   - name: store
+#     description: Access to Petstore orders
```
Comment (peterinfra) [inline ingester.yaml:20]: nitpick: change from petstore

File: ingester.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+              properties:
+                log: 
+                  type: string
+      responses:
+        '200':
+          description: Successful response
+        '400':
+          description: Invalid input
+          headers:
+            X-Custom-Header:
```
Comment (peterinfra) [inline ingester.yaml:47]: what is this for ? Remove if not needed

File: ingester.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+              description: Custom header description
+              schema:
+                type: string
+    get:
```
Comment (peterinfra) [inline ingester.yaml:51]: why would one need this ?

Comment (peterinfra) [reply ingester.yaml:51]: This is an ingester which will not support querying

**Comment by peterinfra** [comment]
You have not updated the PR yet.. Please do so

File: ingester.yaml (hunk 1 above)
Comment (SidDar360) [reply ingester.yaml:5]: What should I change it to?

**Comment by peterinfra** [comment]
reopening it

**Comment by peterinfra** [comment]
reopening

**Pull request closed without merging**